charmtools/build/tactics.py

```diff
@@ -769,6 +769,7 @@ def __init__(self, *args, **kwargs):
         self.tracked = []
         self.previous = []
         self._venv = None
+        self.purge_wheels = False
```

Comment:
I think we should actually make this default, but we can try it out with overrides and flip the switch later.